src/ui/components/Button/index.js

```diff
@@ -9,31 +9,32 @@ import './Button.scss';
 
 export default class Button extends React.Component {
   static propTypes = {
-    appearance: PropTypes.string,
```

Comment:
I actually agree–something like that was originally what I did (though I like your API better!) and now that you brought it up I'm seemingly not the only one. Knowing the classnames is weird but also verbose.

I'll move to your method... I like `<Button action>` a lot and it's easier to have saner defaults then without having to deal with managing `classNames`.

I might do it in another PR so this one doesn't take forever but 👍